### DISCUSSION TITLE
Example configurations for AeromatiC++

### DISCUSSION THREAD

**Discussion by ermarch** [JSBSim-Team/jsbsim#452] [Q&A]
I've collected a number of example aircraft configurations for AeromatiC++ (mostly for testing).
Would it be useful to add it to git like the example aircraft configurations for JSBSim itself?

They are single file text files with an average file size of 1.5kb and range from the default c172p and J3Cub to the B737, B747 as well as the SR-71, Su-15 and F-16.

**Comment by seanmcleod** [comment]
Sure I think it would be useful. 

I presume you're referring to an input parameters log file, `-i` option?

```c++
printf(" -i, --input <file>\t\tRead the input parameters from a log file.\n");
```

So probably in a directory like `utils\aeromatic++\samples`?

**Reply by bcoconni** [reply]
100% agreement with @seanmcleod

**Reply by ermarch** [reply]
Yes, they can be used with the -i option to generated the complete JSBSim configuration.
I'll prepare them and add another pull request.

**Comment by bcoconni** [comment]
The PR has been pulled to `master` and I've also taken this opportunity to add the `aeromatic++` examples to our Windows installer in commit 2f7985ef2149a8836cbf9e292cfff5860c55bedb.
